.github/workflows/CI.yml

```diff
@@ -0,0 +1,193 @@
+name: CI
+on:
+  # always execute docker build when something is pushed to main/master or release-* branches
+  push:
+    branches:
+      - "master"
```

Comment:
maybe we can get rid of this branch name by now :)

Reply:
The basic workflow from the keptn-service-template-go include both, master and main. We can get rid of master in this case.